config/samples/glance_v1beta1_glance_quota.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: glance.openstack.org/v1beta1
+kind: Glance
+metadata:
+  name: glance
+spec:
+  serviceUser: glance
+  containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+  databaseInstance: openstack
+  databaseUser: glance
+  glanceAPIInternal:
+    containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+    debug:
+      service: false
+    preserveJobs: false
+    replicas: 1
+  glanceAPIExternal:
+    containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+    debug:
+      service: false
+    preserveJobs: false
+    replicas: 1
+  secret: osp-secret
+  storageClass: ""
+  storageRequest: 10G
+  quotas:
+    imageSizeTotal: 100
+    imageStageTotal: 100
```

Comment:
Even though it is sample template, I think we should keep default for above two to 1000 (similar to upstream glance default)
https://docs.openstack.org/glance/latest/admin/quotas.html#configuring-glance-for-per-tenant-quotas

Reply:
sure, let me update it to reflect that doc.

Reply:
Done